gensim/corpora/wikicorpus.py

```diff
@@ -620,7 +620,7 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
         Unless a dictionary is provided, this scans the corpus once, to determine its vocabulary.
 
         """
-        self.fname = fname
+        self.input = fname
```

Comment:
Should we rename the param too? That would break API (unless we keep both `input` and `fname`) but maybe a little cleaner. @mpenkov WDYT?